src/ir/var.rs

```diff
@@ -147,6 +201,10 @@ impl ClangSubItemParser for Var {
                         }
                         MacroParsingBehavior::Default => {}
                     }
+
+                    handle_function_macro(&cursor, |left, right| {
```

Comment:
Because `cexpr_tokens()` calls `.tokens()` itself, and because my code needs `ClangToken` instead of `cexpr` tokens, I think the only way to avoid doubly tokenizing things is to refactor `cexpr_tokens`.

I have (locally) a working implementation of
```rust
impl std::convert::TryFrom<ClangToken> for cexpr::token::Token
```
returning `Result<Token, ()>`, but maybe that is a little strange, and maybe it is out of scope for this PR. Would it be better to make `cexpr_tokens` a free function that takes `&[ClangToken]` ?

Reply:
I implemented an `as_cexpr_token()` method on `ClangToken` instead of implementing `TryFrom`.